src/audio/base_fw.c

```diff
@@ -8,6 +8,10 @@
 #include <ipc4/pipeline.h>
```

Comment:
I'm a bit puzzled. System time is maintained by Zephyr. I don't see this PR setting the Zephyr system time, or modifying it. If Zephyr system time is not modified, who would use this information? E.g. log timestamps come from Zephyr uptime, not from SOF IPC applicatin code.

Reply:
Hi, as far as I understand from ACE documentation, this is for passing the information about host system time. It is then used by FW to translate event timestamps e.g. logs to host system time domain. And you are right. Prefix is a little bit unfortunate. Just a leftover from separation into SOF and Zephyr part. I'll update the title and description.